tests/integration/targets/docker_network/tasks/tests/overlay.yml

```diff
@@ -10,17 +10,17 @@
   set_fact:
     dnetworks: "{{ dnetworks + [nname_1] }}"
 
+# Overlay networks require swarm initialization before they'll work
+- name: swarm
```

Comment:
The `-block:` and `always:` part are needed, please add them back so that this task comes at the beginning of the block and the Swarm cleanup in the `always:`. The tests should be the steps after Swarm setup and before `always:`.

Reply:
Done

Reply:
Thanks, looks good now! I'll take another closer look at the PR likely tomorrow, but I think it should be pretty much done (or even completely done :) ).